### DISCUSSION TITLE
Bark

### DISCUSSION THREAD

**Discussion by JoshuaRady** [NGEET/fates#934] [Q&A] [answered]
For a tree stem FATES represents two organs: sapwood and structural wood.  What about the bark?  I should think that the FATES simplification probably includes all the living tissues including phloem, cambium, cork cambium, etc. in sapwood, not just xylem.  Other than the bark thickness used for SPITFIRE I don't think that the outer bark is specifically represented in FATES.  I would like to confirm that this is correct and that the outer bark is not included in another organ.

This is just curiosity.  I don't think the bark representation is something that needs revision or anything.  I was just talking with someone and the C:N:P ratios for the different organs came up, and where the bark was.

**Comment by rosiealice** [comment, marked as answer]
This is indeed my understanding, that there is no specific bark pool, unless something has changed and I missed it. Right @jkshuman @rgknox ??

**Comment by jkshuman** [comment]
That is correct. There is not a specific bark pool.

**Comment by JoshuaRady** [comment]
Thanks for the confirmation @rosiealice & @jkshuman.
